README.adoc

```diff
@@ -27,12 +27,12 @@
 [.hidden]
 NOTE: This repository contains the guide documentation source. To view the guide in published form, view it on the https://openliberty.io/guides/{projectid}.html[Open Liberty website].
 
-Learn how to test a MicroProfile or Jakarta EE application using MicroShed Testing.
+Learn how to use MicroShed Testing to test a MicroProfile or Jakarta EE application.
 
 == What you'll learn
 
-You will start with an existing REST application that runs on OpenLiberty, and use https://microshed.org/microshed-testing/[MicroShed Testing^] 
-to write tests for the application that will exercise the application inside of a Docker container.
+You'll start with an existing REST application that runs on Open Liberty and use https://microshed.org/microshed-testing/[MicroShed Testing^] 
+to write tests for the application that exercise the application inside of a Docker container.
 
 Often times tests may pass in dev/test environments, but fail in production because the application is
 running differently in production than it is in dev/test. Luckily, we can greatly minimize these dev/prod
```

Comment:
Change
"Often times tests may pass in dev/test environments, but fail in production because the application is
running differently in production than it is in dev/test. Luckily, we can greatly minimize these dev/prod
parity issues by testing our application as it runs in the same Docker container we'll be using in production."
To
"Sometimes tests might pass in development and testing (dev/test) environments, but fail in production because the application is running differently in production than it is in dev/test. Fortunately, you can minimize these parity issues between development and production by testing your application in the same Docker container that you'll use in production."

Reply:
addressed